site/mkdocs.yml

```diff
@@ -42,7 +42,6 @@ theme:
   features:
     - navigation.tabs
     - navigation.tabs.sticky
-    - navigation.path
```

Comment:
This is an [MkDocs Material Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) feature. Removing it to avoid confusion.